src/reader-browser.js

```diff
@@ -139,6 +140,22 @@ export class CarReader {
     }
   }
 
+  /**
+   *
```

Comment:
I think it should say something about the fact that no verification occurs with this reader, meaning that block that follows CID may not in fact have that CID.